eng/common/scripts/git-branch-push.ps1

```diff
@@ -27,6 +27,8 @@ param(
     [Parameter(Mandatory = $false)]
     [string] $PushArgs = "",
 
+    [string] $ForkName = "azure-sdk-fork",
```

Comment:
This should really be called remote name as opposed to fork name as the name is a little misleading. We used the name fork because it usually is the azure-sdk fork but in reality this is really just the name of the remote.